diff --git a/.gitignore b/.gitignore
index 74d70d4..7c1070d 100644
--- a/.gitignore
+++ b/.gitignore
@@ -76,4 +76,4 @@ lint/tmp/
 # lint/reports/
 
 app/src/main/assets/app.zip
-signings.gradle
+# signings.gradle
diff --git a/app/build.gradle b/app/build.gradle
index 6922a27..fabab2b 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -56,12 +56,16 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
         debug {
-            applicationIdSuffix ".debug"
+            signingConfig signingConfigs.debug
 
-            minifyEnabled false
+            // applicationIdSuffix ".debug"
 
-            resValue "string", "app_name", "SiYuan-Debug"
-            resValue "string", "app_package_name", "org.b3log.siyuan.debug"
+            // minifyEnabled false
+            minifyEnabled true
+            shrinkResources true
+
+            resValue "string", "app_name", "SiYuan" // SiYuan-Debug
+            resValue "string", "app_package_name", "org.b3log.siyuan" // org.b3log.siyuan.debug
 
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
diff --git a/signings.templates.gradle b/signings.gradle
similarity index 67%
rename from signings.templates.gradle
rename to signings.gradle
index d3e168c..59691d3 100644
--- a/signings.templates.gradle
+++ b/signings.gradle
@@ -9,5 +9,12 @@ android {
       keyAlias "androiddebugkey"   // Alias
       keyPassword "android"        // Alias 密码
     }
+
+    debug {
+      storeFile = file("../keystore/debug.jks")
+      storePassword System.getenv("KEYSTORE_PASSWORD")
+      keyAlias System.getenv("KEY_ALIAS")
+      keyPassword System.getenv("KEYSTORE_PASSWORD")
+    }
   }
 }
