diff --git a/kernel/conf/account.go b/kernel/conf/account.go
index 5e3b51fb8..d4f4e00de 100644
--- a/kernel/conf/account.go
+++ b/kernel/conf/account.go
@@ -23,7 +23,7 @@ type Account struct {
 
 func NewAccount() *Account {
 	return &Account{
-		DisplayTitle: true,
-		DisplayVIP:   true,
+		DisplayTitle: false,
+		DisplayVIP:   false,
 	}
 }
diff --git a/kernel/conf/sync.go b/kernel/conf/sync.go
index 1f23e9642..9e002cbd7 100644
--- a/kernel/conf/sync.go
+++ b/kernel/conf/sync.go
@@ -37,8 +37,8 @@ func NewSync() *Sync {
 		Enabled:             false,
 		Perception:          false,
 		Mode:                1,
-		GenerateConflictDoc: false,
-		Provider:            ProviderSiYuan,
+		GenerateConflictDoc: true,
+		Provider:            ProviderWebDAV,
 		Interval:            30,
 	}
 }
diff --git a/kernel/conf/system.go b/kernel/conf/system.go
index 52c910688..ac65790b4 100644
--- a/kernel/conf/system.go
+++ b/kernel/conf/system.go
@@ -55,6 +55,7 @@ func NewSystem() *System {
 		Name:               util.GetDeviceName(),
 		KernelVersion:      util.Ver,
 		NetworkProxy:       &NetworkProxy{},
+		DisableGoogleAnalytics: true,
 		DownloadInstallPkg: true,
 	}
 }
diff --git a/kernel/model/updater.go b/kernel/model/updater.go
index 747972b63..06246fb07 100644
--- a/kernel/model/updater.go
+++ b/kernel/model/updater.go
@@ -140,18 +140,18 @@ func getUpdatePkg() (downloadPkgURLs []string, checksum string, err error) {
 	}
 	pkg := "siyuan-" + ver + "-" + suffix
 
-	b3logURL := "https://release.b3log.org/siyuan/" + pkg
-	liuyunURL := "https://release.liuyun.io/siyuan/" + pkg
-	githubURL := "https://github.com/siyuan-note/siyuan/releases/download/v" + ver + "/" + pkg
+	// b3logURL := "https://release.b3log.org/siyuan/" + pkg
+	// liuyunURL := "https://release.liuyun.io/siyuan/" + pkg
+	githubURL := "https://github.com/golabol/my-siyuan/releases/download/v" + ver + "/" + pkg
 	ghproxyURL := "https://ghfast.top/" + githubURL
 	if util.IsChinaCloud() {
-		downloadPkgURLs = append(downloadPkgURLs, b3logURL)
-		downloadPkgURLs = append(downloadPkgURLs, liuyunURL)
+		// downloadPkgURLs = append(downloadPkgURLs, b3logURL)
+		// downloadPkgURLs = append(downloadPkgURLs, liuyunURL)
 		downloadPkgURLs = append(downloadPkgURLs, ghproxyURL)
 		downloadPkgURLs = append(downloadPkgURLs, githubURL)
 	} else {
-		downloadPkgURLs = append(downloadPkgURLs, b3logURL)
-		downloadPkgURLs = append(downloadPkgURLs, liuyunURL)
+		// downloadPkgURLs = append(downloadPkgURLs, b3logURL)
+		// downloadPkgURLs = append(downloadPkgURLs, liuyunURL)
 		downloadPkgURLs = append(downloadPkgURLs, githubURL)
 		downloadPkgURLs = append(downloadPkgURLs, ghproxyURL)
 	}
