diff --git a/app/appearance/langs/en_US.json b/app/appearance/langs/en_US.json
index 240ec4032..9f93de009 100644
--- a/app/appearance/langs/en_US.json
+++ b/app/appearance/langs/en_US.json
@@ -1196,7 +1196,7 @@
   "about18": "When using a non-loopback address (<code class=\"fn__code\">127.0.0.1</code>/<code class=\"fn__code\">[::1]</code>) please set the Access authorization code first",
   "checkUpdate": "Check update",
   "currentVer": "Current version",
-  "downloadLatestVer": "<a href=\"https://b3log.org/siyuan/en/download.html\", target=\"_blank\">Download the latest version</a>",
+  "downloadLatestVer": "<a href=\"https://github.com/golabol/my-siyuan/releases/latest\", target=\"_blank\">Download the latest version</a>",
   "themeLight": "Light",
   "themeDark": "Dark",
   "themeOS": "Follow the system",
diff --git a/app/electron/error.html b/app/electron/error.html
index 4dc3cd3d0..8b7b4fa83 100644
--- a/app/electron/error.html
+++ b/app/electron/error.html
@@ -150,7 +150,7 @@
     </div>
     <div class="feedback">
         <span><a href="https://liuyun.io/article/1686530886208" target="_blank">Feedback and support</a></span>
-        <span><a href="https://b3log.org/siyuan/en/download.html" target="_blank">Download the latest version</a></span>
+        <span><a href="https://github.com/golabol/my-siyuan/releases/latest" target="_blank">Download the latest version</a></span>
     </div>
     <div class="fn__flex-1"></div>
 </div>
diff --git a/kernel/conf/sync.go b/kernel/conf/sync.go
index 1f23e9642..9e002cbd7 100644
--- a/kernel/conf/sync.go
+++ b/kernel/conf/sync.go
@@ -37,8 +37,8 @@ func NewSync() *Sync {
 		Enabled:             false,
 		Perception:          false,
 		Mode:                1,
-		GenerateConflictDoc: false,
-		Provider:            ProviderSiYuan,
+		GenerateConflictDoc: true,
+		Provider:            ProviderWebDAV,
 		Interval:            30,
 	}
 }
diff --git a/kernel/conf/system.go b/kernel/conf/system.go
index 52c910688..ac65790b4 100644
--- a/kernel/conf/system.go
+++ b/kernel/conf/system.go
@@ -55,6 +55,7 @@ func NewSystem() *System {
 		Name:               util.GetDeviceName(),
 		KernelVersion:      util.Ver,
 		NetworkProxy:       &NetworkProxy{},
+		DisableGoogleAnalytics: true,
 		DownloadInstallPkg: true,
 	}
 }
diff --git a/kernel/model/updater.go b/kernel/model/updater.go
index 747972b63..06246fb07 100644
--- a/kernel/model/updater.go
+++ b/kernel/model/updater.go
@@ -140,18 +140,18 @@ func getUpdatePkg() (downloadPkgURLs []string, checksum string, err error) {
 	}
 	pkg := "siyuan-" + ver + "-" + suffix
 
-	b3logURL := "https://release.b3log.org/siyuan/" + pkg
-	liuyunURL := "https://release.liuyun.io/siyuan/" + pkg
-	githubURL := "https://github.com/siyuan-note/siyuan/releases/download/v" + ver + "/" + pkg
+	// b3logURL := "https://release.b3log.org/siyuan/" + pkg
+	// liuyunURL := "https://release.liuyun.io/siyuan/" + pkg
+	githubURL := "https://github.com/golabol/my-siyuan/releases/download/v" + ver + "/" + pkg
 	ghproxyURL := "https://ghfast.top/" + githubURL
 	if util.IsChinaCloud() {
-		downloadPkgURLs = append(downloadPkgURLs, b3logURL)
-		downloadPkgURLs = append(downloadPkgURLs, liuyunURL)
+		// downloadPkgURLs = append(downloadPkgURLs, b3logURL)
+		// downloadPkgURLs = append(downloadPkgURLs, liuyunURL)
 		downloadPkgURLs = append(downloadPkgURLs, ghproxyURL)
 		downloadPkgURLs = append(downloadPkgURLs, githubURL)
 	} else {
-		downloadPkgURLs = append(downloadPkgURLs, b3logURL)
-		downloadPkgURLs = append(downloadPkgURLs, liuyunURL)
+		// downloadPkgURLs = append(downloadPkgURLs, b3logURL)
+		// downloadPkgURLs = append(downloadPkgURLs, liuyunURL)
 		downloadPkgURLs = append(downloadPkgURLs, githubURL)
 		downloadPkgURLs = append(downloadPkgURLs, ghproxyURL)
 	}
