diff --git a/kernel/api/setting.go b/kernel/api/setting.go
index e5765d64a..e56e9efa5 100644
--- a/kernel/api/setting.go
+++ b/kernel/api/setting.go
@@ -593,6 +593,28 @@ func getPublish(c *gin.Context) {
 	}
 }
 
+// func getCloudUser(c *gin.Context) {
+// 	ret := gulu.Ret.NewResult()
+// 	defer c.JSON(http.StatusOK, ret)
+
+// 	if !model.IsAdminRoleContext(c) {
+// 		return
+// 	}
+
+// 	arg, ok := util.JsonArg(c, ret)
+// 	if !ok {
+// 		return
+// 	}
+
+// 	t := arg["token"]
+// 	var token string
+// 	if nil != t {
+// 		token = t.(string)
+// 	}
+// 	model.RefreshUser(token)
+// 	ret.Data = model.Conf.GetUser()
+// }
+
 func getCloudUser(c *gin.Context) {
 	ret := gulu.Ret.NewResult()
 	defer c.JSON(http.StatusOK, ret)
@@ -601,18 +623,38 @@ func getCloudUser(c *gin.Context) {
 		return
 	}
 
-	arg, ok := util.JsonArg(c, ret)
-	if !ok {
-		return
-	}
+	user := &conf.User{
+			UserId: "0",
+			UserName: "SiYuan",
+			UserAvatarURL: "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
+			UserHomeBImgURL: "",
+			UserTitles: []*conf.UserTitle{},
+			UserIntro: "",
+			UserNickname: "",
+			UserCreateTime: "20201231 00:00:00",
+			UserSiYuanProExpireTime: -1,
+			UserToken: "b8e4c9d1-3f7a-4e2b-9b8a-2e4c1a8f6c3d",
+			UserTokenExpireTime: "34359738368",
+			UserSiYuanRepoSize: 0,
+			UserSiYuanPointExchangeRepoSize: 0,
+			UserSiYuanAssetSize: 0,
+			UserTrafficUpload: 0,
+			UserTrafficDownload: 0,
+			UserTrafficAPIGet: 0,
+			UserTrafficAPIPut: 0,
+			UserTrafficTime: 0,
+			UserSiYuanSubscriptionPlan: 0,
+			UserSiYuanSubscriptionStatus: 0,
+			UserSiYuanSubscriptionType: 1,
+			UserSiYuanOneTimePayStatus: 1,
+	}
+
+	model.Conf.User = user
+	data, _ := gulu.JSON.MarshalJSON(user)
+	model.Conf.UserData = util.AESEncrypt(string(data))
+	model.Conf.Save()
 
-	t := arg["token"]
-	var token string
-	if nil != t {
-		token = t.(string)
-	}
-	model.RefreshUser(token)
-	ret.Data = model.Conf.GetUser()
+	ret.Data = user
 }
 
 func logoutCloudUser(c *gin.Context) {
